### PR TITLE
Announcing Scala.js 0.6.18.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by sjrd** [comment]
To be announced tomorrow (Wednesday).

File: tutorial/basic/index.md [hunk 1]
```diff
@@ -130,7 +128,7 @@ To load and launch the created JavaScript, you will need an HTML file. Create th
 
 The script tag simply includes the generated code (attention, you might need to adapt the Scala version from `2.12` to `2.10` or `2.11` here if you are using Scala 2.10.x or Scala 2.11.x instead of 2.12.x).
 
-Since `TutorialApp` extends `js.JSApp`, and we have set `scalaJSUseMainModuleInitializer := true` in the build, the `TutorialApp.main()` method is automatically called at the end of the `-fastopt.js` file.
+Since `TutorialApp` extends `js.JSApp`, and we have set `scalaJSUseMainModuleInitializer := true` in the build, the `TutorialApp.main()` method is automatically called at the end of the `-fastopt.js` file (with an empty array as argument).
```
Comment (gzm0) [inline tutorial/basic/index.md]: It doesn't extend `js.JSApp` anymore :)

Comment (sjrd) [inline tutorial/basic/index.md]: Ah! Nice catch.

Comment (sjrd) [inline tutorial/basic/index.md]: Fixed.

**Pull request merged**